src/rapier2d-wrapper/src/lib.rs

```diff
@@ -987,7 +1014,22 @@ pub extern "C" fn shape_create_circle(radius : f32) -> Handle {
 
 #[no_mangle]
 pub extern "C" fn shape_create_capsule(half_height : f32, radius : f32) -> Handle {
-	let shape = SharedShape::capsule_y(half_height, radius);
+	let top_circle = SharedShape::ball(radius);
```

Comment:
Is there any reason not to add a "useHack" parameter right now to default to false, which would allow the hack to be used only for a CharacterBody in c++?